ios/MarkdownCommitHook.mm

```diff
@@ -141,14 +141,12 @@
                     stateData.attributedStringBox);
 
             // Handles the first render, where the text stored in props is
-            // different than the one stored in state The one in state is empty,
-            // while the one in props is passed from JS If we don't update the
-            // state here, we'll end up with a one-default-line-sized text
-            // input. A better condition to do that can be probably chosen, but
-            // this seems to work
+            // different than the one stored in state. The one in state is empty,
+            // while the one in props is passed from JS. If we don't update the
+            // state here, we'll end up with a one-default-line-sized text input
             auto plainString =
                 std::string([[nsAttributedString string] UTF8String]);
-            if (plainString != textInputProps.text) {
+            if (textInputState.getRevision() == State::initialRevisionValue && plainString != textInputProps.text) {
```

Comment:
I think we can even skip comparing state text with props text, but it's been a while since I've touched this code so I'm not sure. We can merge this as-is to fix the issue and investigate it further later.

Reply:
> I think we can even skip comparing state text with props text,

Yeah I was wondering the same. I think the only case where state == props on first render is when the input is empty 🤔 (but not 100% sure either)